lib/rules.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Rules do
+  def winning_combos(board) do
+    Enum.into(Board.row(board), Board.column(board))
+    |> Enum.into(Board.diagonal(board))
+  end
+
+  def game_in_progress?(board) do
+    (win?(board) == false && draw?(board) == false)
+    || (win?(board) == nil && draw?(board) == false)
+  end
+
+  def win?(board) do
+    if board == Board.empty_board() do
+      false
+    else
+      Enum.find_value(winning_combos(board), fn(x) -> identical_player_symbol?(x) end)
+    end
+  end
+
+  def draw?(board) do
+    if board == Board.empty_board() do
+      false
+    else
+      Board.is_full?(board) && win?(board) == nil
+    end
+  end
+
+  defp identical_player_symbol?(enumerable) do
+    Enum.uniq(enumerable) |> Kernel.length == 1 && Enum.uniq(enumerable) != [:empty]
```

Comment:
When you use the |> symbol, put it on the next line unless the command is particularly short.

Bad:

some_command_goes_here() |> this_is_a_really_long_command()

Good:

some_command_goes_here() 
|> this_is_a_really_long_command()